packages/graphile-build-pg/src/plugins/introspectionQuery.js

```diff
@@ -379,13 +380,14 @@ with
       pg_catalog.pg_index as idx
       inner join pg_catalog.pg_class idx_more on (idx.indexrelid = idx_more.oid)
       inner join class on (idx.indrelid = class.id)
+      inner join pg_class as idxclass on (idx.indexrelid = idxclass.oid)
       inner join pg_catalog.pg_namespace as nsp on (nsp.oid = idx_more.relnamespace)
+      inner join pg_am as am on (am.oid = idxclass.relam)
       left join pg_catalog.pg_description as dsc on dsc.objoid = idx.indexrelid and dsc.objsubid = 0 and dsc.classoid = 'pg_catalog.pg_class'::regclass
     where
       idx.indislive is not false and
       idx.indisexclusion is not true and -- exclusion index
       idx.indcheckxmin is not true and -- always valid?
-      not (string_to_array(idx.indkey::text, ' ')::int2[] operator(pg_catalog.@>) ARRAY[0::int2]) and -- no expressions
```

Comment:
Please add something to the tests to ensure this doesn't cause issues - currently our test suite does not contain indexes with expressions so we cannot be sure this doesn't cause issues.